west.yml

```diff
@@ -38,7 +38,7 @@ manifest:
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
       west-commands: scripts/west-commands.yml
-      revision: 4ebad02e3e270169220a9959f8c9ea6ba88f2e28
+      revision: pull/207/head
```

Comment:
The Zephyr SHA in the manifest file on master is now ahead of this PR, please remove this change.